hypex/transformers/filters.py

```diff
@@ -31,41 +31,44 @@ def __init__(
         super().__init__(key=key)
         self.lower_bound = lower_bound
         self.upper_bound = upper_bound
+        self.type_filter: bool = True
 
-    def calc(
-        self,
+    @staticmethod
+    def _inner_function(
         data: Dataset,
-        target_roles: Union[ABCRole, Iterable[ABCRole]] = FeatureRole(),
-        type_filter: bool = True,
+        target_cols: Optional[FieldKeyTypes] = None,
+        lower_bound: Optional[float] = None,
+        upper_bound: Optional[float] = None,
     ) -> Dataset:
-        target_roles = super()._list_unification(target_roles)
-        if type_filter:
-            addressable_columns = data.search_columns(
-                roles=target_roles, search_types=[float, int, bool]
-            )
-        else:
-            addressable_columns = data.search_columns(target_roles)
-        for column in addressable_columns:
+        target_cols = super(CVFilter, CVFilter)._list_unification(target_cols)
```

Comment:
you can write just super()._list_unification

Reply:
By some reason it does not work, but anyways it is replaced with adapter method.